docker/api/network.py

```diff
@@ -33,6 +37,7 @@ def remove_network(self, net_id):
         url = self._url("/networks/{0}", net_id)
         res = self._delete(url)
         self._raise_for_status(res)
+        return True
```

Comment:
No real reason that I can remember. I may have been on the fence at the time about returning `False` if the operation doesn't complete in non-exceptional contexts, but I don't think we have any of those, so it's a moot point.